src/generate/templates/ping-device-ping1d.h.in

```diff
@@ -51,29 +51,23 @@ public:
     bool {{msg}}({% for field in messages["set"][msg].payload %}{{generator.get_type_string(field.type)}} {{field.name}}, {% endfor %}bool verify = true);
 
 {% endfor %}
-{% set propnames=[] %}
-{% set proplist=[] %}
-{% for msg in messages["get"] %}
+
+{% for msg in messages["get"]|sort %}
 {% if not messages["get"][msg].deprecated %}
+    //! {{messages["get"][msg].description}}
```

Comment:
what is the `!` for?

Reply:
http://www.doxygen.nl/manual/docblocks.html